tests/test_association_task.py

```diff
@@ -192,24 +285,33 @@ def _run_association_and_retrieve_objects(self, create_objects=False):
             start_id=10,
             scatter_arcsec=-1)
         for dia_source in dia_sources:
-            dia_source["psFlux"] = 10000.
-            dia_source["psFluxErr"] = 100.
+            dia_source['ccdVisitId'] = \
+                self.exposure.getInfo().getVisitInfo().getExposureId()
+            dia_source["midPointTai"] = \
+                self.exposure.getInfo().getVisitInfo().getDate().get(system=dafBase.DateTime.MJD)
+            dia_source["psFlux"] = 10000 / self.flux0
+            dia_source["psFluxErr"] = \
+                np.sqrt((100 / self.flux0) ** 2 + (10000 * self.flux0_err / self.flux0 ** 2) ** 2)
+            dia_source["apFlux"] = 10000 / self.flux0
+            dia_source["apFluxErr"] = \
+                np.sqrt((100 / self.flux0) ** 2 + (10000 * self.flux0_err / self.flux0 ** 2) ** 2)
+            dia_source["filterName"] = self.exposure.getFilter().getName()
+            dia_source["filterId"] = self.exposure.getFilter().getId()
+            dia_source["x"] = 0
+            dia_source["y"] = 0
+            dia_source["snr"] = 10
 
         assoc_config = AssociationConfig()
-        assoc_config.level1_db.value.db_name = self.db_file
-        assoc_config.level1_db.value.filter_names = self.filter_names
         assoc_task = AssociationTask(config=assoc_config)
 
-        assoc_task.run(dia_sources, self.exposure)
-        assoc_task.level1_db.close()
+        ppdb = Ppdb(config=self.ppdbConfig,
+                    afw_schemas=dict(DiaObject=make_dia_object_schema(),
+                                     DiaSource=make_dia_source_schema()))
+        ppdb._schema.makeSchema()
```

Comment:
Given the discussion on lsst/dax_ppdb#5, this line should be `ppdb.makeSchema()` but otherwise doesn't need to be changed.

Reply:
Does the new make_schema function satisfy this?

Reply:
If you really meant `_make_ppdb`, then mostly yes. Did you mean to leave off the `drop=True` bit?